### DISCUSSION TITLE
Is Zen lost its identity and becoming arc wrapper on firefox?

### DISCUSSION THREAD

**Discussion by RABINGG** [zen-browser/desktop#2830] [General]
@mauro-balades 
I ditched Arc for Zen, and I gotta say, it's got some sick features like compact mode, vertical tabs with hover and minimize options, and a UI that's super customizable. But, it seems like the Zen community is trying to turn it into just an Arc clone. Earlier I also used to request features from Arc to come to Zen, and some of them did, but they got a Zen twist. Now, it feels like cut, copy and paste. TBH, Workspace theme change success alert that's totally not Zen-vibe just CCP. Is this gonna be the future of Zen?

**Comment by milyrouge** [comment]
I'm also in the process of ditching Arc for Zen, as are many others with the recent announcement of the deprioritisation of Arc (I'm a long-time Firefox user so love the idea of getting Firefox with the next-gen UI à la Zen and Arc). This is a great opportunity for Zen to take advantage of getting a lot of users already bought into a philosophy similar to Zen. I say, deliver the existing Zen roadmap, steal with pride when it comes to those Arc-like features that are stopping the migration, get those users across and then add the Zen spin when it's time and resources allow!

**Comment by mr-cheffy** [comment]
Right now, the priority is getting all features and fixes as fast as possible. After that, we can start making our own identity

**Reply by rubaboo** [reply]
What features though? Those Arc defectors are making a lot of noise here in these discussions. I came to Zen from Firefox/Librewolf and don't really want any of the Arc features. Some I dislike, most I just never knew and don't care. This is exactly what OP is asking: what is to become of Zen if it doesn't have it's own identity? A clone of Arc? An awful lot of requests here seem to be steering you in that direction.

**Comment by KakouMoses** [comment]
Hi, I'm new and coming from Arc to Zen, and I hope they'll take advantage of the situation as fast as possible...

**Comment by ArikRahman** [comment]
I like that it is a arc wrapper on firefox, especially because I use Linux, and couldn't have an arc-like environment otherwise. For me, this is the greatest strength, and I wouldn't have it any other way.

**Comment by mikedlr** [comment]
What is Zen identity? For me:

* neat dark look and black icons (most visible but least important)
* left hand sidebar with vertical tabs
* full support for most Firefox plugins 
* workspaces with a direct mapping to containers allowing effective working
* cleverly pinned tabs with key web applications but ability to smoothly link into unpinned tabs *in the same browser* for efficient working
* useful use of colours and icons so I can see what's going on. 
* native inclusion of and better integration things like containers that are mostly managed with extensions on Firefox.

I don't see any of that being lost though I see developers doing a "move fast and break things" to make it all fit together better and I want to help them because I currently use two "browsers" (Firefox and Rambox) for what would only need one if Zen worked properly as described.

**Reply by ArikRahman** [reply]
I completely agree with points mentioned, and I'd like to add my own take on the identity. I personally love the workflow of tiling managers and use Zen within Hyprland. With the additions of workspaces, I can have workspaces within my workspaces! 

Zen pursuing this route will be a boon for all, although I agree, the pace comes at the cost of breaking changes at times.

**Comment by rubaboo** [comment]
All new features that I tried were half-baked.
* Workspaces button turned inactive and invisible.
* Essentials appeared in every window, but have to be removed from each window separately.
* Disabling a theme rarely reverts the look to the previous without restarting the browser.

I like "Beautifully designed, privacy-focused", but feature packing conveyor needs some quality control.

**Reply by mikedlr** [reply]
I think that is a slightly separate issue. There's clearly some stuff that could be added to the software creation process. I'd say that if some basic features are breaking and it's being released then it would really help the developers if automated tests that go through each of those features were added to the build process. 

Getting the first test or two working would be a great way to help the process for someone that knew how.

**Comment by rubaboo** [comment]
The latest development is troubling. "huge amounts of bugs due to a firefox upgrade"? That just validates mine and OPs concerns. Sad.

**Comment by mikedlr** [comment]
I made [a discussion about Rambox](https://github.com/zen-browser/desktop/discussions/4006) which is a "Workspace simplifier", in other words a partial browser that only deals with web applications. I think that if Zen thought about what Rambox does better than Arc and how to do that in an actual browser, that would be a way to get  a more unique identity.

**Comment by GJRobert** [comment]
I am satisfied for all Arc-like UI improvements so far.
I will keep like Zen as it adds new features while maintaining such "all-hidden" experience.

**Comment by rubaboo** [comment]
I am a long-time user of Netscape Navigator and then Firefox and it's derivatives. I was hoping Zen would stand out among them as the one with a modern good look and a lot of customizeability (not a word apparently). Unfortunately, as of now, it stands out for different reasons. I could try to expand on that, but these two have already covered everything that really needed to be said:

the OP:
> the Zen community is trying to turn it into just an Arc clone.

`@typhon88`(disabling a mention since IDK if it's the same person) over on [Reddit](https://www.reddit.com/r/zen_browser/comments/1h1vfky/release_cycle/):
> Id venture a guess that most people here dont know anything about software development and are just barking out orders cause they need their specific ARC feature working today! Trying to make too many changes for a minor point release each week with a small unpaid development team will ultimately lead to an unstable product which over time will make people look elsewhere. No one wants that.

On my part, I tried to be helpful by suggesting that people try using Firefox extensions instead. They often provide the same features and a lot more. But I ended up alienating people who would not have it any other way.
